ui/src/components/settings/sub_pages/audio.rs

```diff
@@ -16,6 +21,59 @@ pub fn AudioSettings(cx: Scope) -> Element {
         div {
             id: "settings-audio",
             aria_label: "settings-audio",
+            SettingSection {
+                section_label: get_local_text("settings-audio.input-device"),
+                section_description: get_local_text("settings-audio.input-device-description"),
+                no_border: true,
+                Select {
+                    initial_value: "Default".into(),
```

Comment:
"Default" should probably be replaced by a call to `get_local_text()`

Reply:
These are just placeholders for when you wanted to hook things up. I just put them there so things were not empty.